app/actions/CompanyActions.ts

```diff
@@ -8,11 +8,12 @@ import {
   joblistingsSchema,
 } from 'app/reducers';
 import createQueryString from 'app/utils/createQueryString';
-import { startSubmit, stopSubmit } from 'redux-form';
-import { push } from 'connected-react-router';
 import type { Thunk } from 'app/types';
 import { addToast } from 'app/actions/ToastActions';
```

Comment:
This does not sort as well though. You'd preferably want `import { addToast } from 'app/actions/ToastActions';` to come next to `import callAPI from 'app/actions/callAPI';`

Reply:
Hmm, true. I can try enabeling alphabetical sorting

Reply:
@ivarnakken What do you think now. Now all imports in each group are sorted alphabetically